### ISSUE TITLE
Does not work on node.js v0.10.25

### ISSUE THREAD

**Issue by narmontas** [wmluke/gulp-inline-angular-templates#1]
Your example does not work, i get TypeError.

Looking at your module.exports function i see variables `target` and `options`,
but options arrives to `target` variable, then line

```
fs.src(path.join(process.cwd(), target))
```

errors with `target` must be string.

**Comment by jameswyse** [comment]
Ran in to the same problem, after looking through the code I worked out that the documentation is simply wrong. Try this:

``` javascript
gulp.src('src/templates/*.html')
  .pipe(inlineTemplates('src/index.html', options))
  .pipe(gulp.dest('dist'));
```

**Comment by barronj** [comment]
Please fix the documentation. I was going to give up and use gulp-angular-templatecache instead of this even though I philosophically like keeping my HTML together instead of embedding it into Javascript. Seeing this thread was the only thing that changed my mind and stick with it.

**Comment by wmluke** [comment]
Fixed with PR #2
